<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIdioma</class>
 <widget class="QDialog" name="DialogIdioma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Idiomas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #A4B8C4</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listaIdioma"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Doble click
para seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newIdioma">
       <property name="placeholderText">
        <string>Nuevo idioma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Agregar 
Idioma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>98</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
